src/kernels/cosine.jl

```diff
@@ -0,0 +1,17 @@
+"""
+`CosineKernel()`
+The cosine kernel is a stationary kernel for a sinusoidal with period p in 1d given by
```

Comment:
According to the [Julia documentation](https://docs.julialang.org/en/latest/manual/documentation/), this should be formatted as
```suggestion
    CosineKernel

The cosine kernel is a stationary kernel for a sinusoidal with period p in 1d given by
```